tools/python_api/src_py/query_result.py

```diff
@@ -326,32 +326,28 @@ def encode_rel_id(rel: dict[str, Any]) -> tuple[int, int]:
                     continue
                 column_type, _ = properties_to_extract[i]
                 if column_type == Type.NODE.value:
-                    _id = row[i]["_id"]
-                    nodes[(_id["table"], _id["offset"])] = row[i]
-                    table_to_label_dict[_id["table"]] = row[i]["_label"]
+                    id = row[i]["_id"]
```

Comment:
I would not remove the unserscores from this function. Specifically not from `_id`. Python has a built-in `id` function, removing `_` will mask it, which is bad.